### ISSUE TITLE
Inconsistency between checks 1663 & 1669

### ISSUE THREAD

**Issue by Richard-Fowle** [iho-ohi/ENCWG-S58-ENC-Validation-Checks#41]
From Herman Schouten van der Velden - Dienst der Hydrografie

Checks 1663 and 1669 provide the check logic for more or less the same encoding.
Is it intended that for WRECKS this check is error level WARNING and for OBSTRN the error level is ERROR?
Further focusing on WATLEV 1 (partly submerged) or 2 (always dry) the HEIGHT for WRECKS is optional while for OBSTRN this must be Present.
Aligning the order of the columns between the checks could maybe be a good suggestion.
So in all not major issues and maybe already noticed and marked up for new versions of S-58, or perhaps there is a logic to the difference between the two features.

[Checks 1663 & 1669 - inconsistencies.docx](https://github.com/iho-ohi/ENCWG-S58-ENC-Validation-Checks/files/11120739/Checks.1663.1669.-.inconsistencies.docx)

**Comment by Richard-Fowle** [comment]
I agree with the observations from Herman and propose downgrading check 1669 to a warning, amending 'present' to 'optional' for height and switching the columns for height and TECSOU/HEIGHT. 
[Proposed changes to Checks 1663 & 1669.docx](https://github.com/iho-ohi/ENCWG-S58-ENC-Validation-Checks/files/11120868/Proposed.changes.to.Checks.1663.1669.docx)

**Comment by Christian-Shom** [comment]
Agree with Richard's proposed changes.
For check 1663, "value" is missing in the top third column.

**Comment by FrankHippmann** [comment]
It looks as though the proposal for making OBSTRN/HEIGHT optional (when WATLEV = 1 or 2) did not make it through to the version at https://iho.int/uploads/user/Services%20and%20Standards/HSSC/HSSC16/S-58%20Edition%208.0.0_redline_20240322.docx. Since it is clearly the intent that HEIGHT is not mandatory for OBSTRN, I propose to reopen this issue.
